### PR TITLE
Add human readable days routine example

### PR DESCRIPTION
**Pull request by mosabua**
## Description

- Identical to the routine shown in the SQL training

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Review by nineinchnick — APPROVED** [review]

**Comment by mosabua** [comment]
Hahah .. you are fast @nineinchnick

**Comment by nineinchnick** [comment]
I watch the #github channel on Slack :-)

File: docs/src/main/sphinx/routines/examples.md [hunk 1]
```diff
@@ -628,8 +693,8 @@ The preceding query produces the following output:
 
 Trino already has a built-in `bar()` [color function](/functions/color), but
 it's using ANSI escape codes to output colors, and thus is only usable for
-displaying results in a terminal. The following example shows a similar
-function, that only uses ASCII characters.
+displaying results in a terminal. The following example shows a similar routine,
```
Comment (findinpath) [inline docs/src/main/sphinx/routines/examples.md:696]: nit: separate commit

Comment (mosabua) [reply docs/src/main/sphinx/routines/examples.md:696]: Not worth it .. same page anyway

**Comment by mosabua** [comment]
@electrum @martint ?

**Review by dain — APPROVED** [review]

**Comment by mosabua** [comment]
Thank you for the review @dain

**Pull request merged**